### ISSUE TITLE
GET /V1/inventory/low-quantity-notification/:sourceCode/:sku allows you to specify invalid parameters

### ISSUE THREAD

**Issue by keharper** [magento/inventory#1720]
### Preconditions

1. Install MSI 2.3-develop

### Steps to reproduce

1. Create a stock and source, and assign a product to a source. Define the quantity available from the source.
2. Define the notify quantity for the product.

```
POST /V1/inventory/low-quantity-notification

{
  "sourceItemConfigurations": [
    {
      "source_code": "reno_wh",
      "notify_stock_qty": 10,
      "sku": "sp2"
    }
  ]
}
```
3. Run `GET /V1/inventory/low-quantity-notification/:sourceCode/:sku`, but misspell the `sourceCode` parameter (change the underbar to a dash in this case):

`GET /V1/inventory/low-quantity-notification/reno-wh/sp2`

### Expected result

An error message stating no such sourceCode exists

### Actual result

A response containing a system-defined value for `notify_stock_qty`.

```
{
  "source_code": "reno-wh",
  "notify_stock_qty": 1,
  "sku": "sp2"
}
```

The same `notify_stock_qty` value is returned if you specify an invalid sku.

**Comment by larsroettig** [comment]
Hi @maghamed,
this is a Feature, not a Bug it was implemented in #237. 

What is the correct behavior we should throw an Exception?

For I think is not correct to return a default config on any case because in the database we don't have an entry for this item. 

Maybe it better to a param to get the default config. 

What is your opinion?


best regards,

Lars

**Comment by larsroettig** [comment]
Hi @maghamed, 
if you describe the expected behaviour I can implement this Issue.  

Currently, it is not a bug it was implemented in #237.

best regards, 

Lars
